compiler/src/dotty/tools/repl/ParseResult.scala

```diff
@@ -132,8 +132,6 @@ object ParseResult {
       case _ =>
         implicit val ctx: Context = state.context
 
-        val source = SourceFile.virtual(str.REPL_SESSION_LINE + (state.objectIndex + 1), sourceCode)
-
         val reporter = newStoreReporter
         val stats = parseStats(sourceCode)(state.context.fresh.setReporter(reporter).withSource(source))
```

Comment:
The parameter `sourceCode` is never used inside `parseStats`, you could remove it

Reply:
Nice catch, removed it